src/uthash.h

```diff
@@ -115,6 +138,16 @@ typedef unsigned char uint8_t;
 /* calculate the hash handle from element address elp */
 #define HH_FROM_ELMT(tbl,elp) ((UT_hash_handle *)(((char*)(elp)) + ((tbl)->hho)))
 
+#define HASH_ROLLBACK_BKT(hh, head, itemptrhh)                                   \
+do {                                                                             \
+  struct UT_hash_handle *_hd_hh_item = (itemptrhh);                              \
+  unsigned _hd_bkt;                                                              \
+  HASH_TO_BKT(_hd_hh_item->hashv, (head)->hh.tbl->num_buckets, _hd_bkt);         \
+  (head)->hh.tbl->buckets[_hd_bkt].count++;                                      \
```

Comment:
Shouldn't this be `count--`, not `count++`?  Please add a unit-test that would have caught this bug (assuming it is a bug), before fixing it.
Also style nit: the backslashes `\` on lines 147-148 should be aligned.